packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -25,6 +45,36 @@ declare global {
        * @category Containers util
        */
       startWildflyContainer(): Chainable<unknown>;
+      /**
+       * Start a Wildfly container. This typical command which needs to executed in before method in every TC.
+       *
+       * This container will have a secured management API.
+       *
+       * @category Containers util
+       *
+       * @param portOffset - optional parameter to set offset for all used ports.
+       *
+       */
+      startWildflyContainerSecured(): Chainable<unknown>;
+      /**
+       * Start a SSO container. This command which needs to executed in before method to start and configure SSO.
```

Comment:
> This command which needs to executed in before method to start and configure SSO.

Do you mean something like

This command typically needs to be executed in `before` method of a test spec.